src/Bridge/Doctrine/Orm/Filter/OrderFilter.php

```diff
@@ -103,7 +103,11 @@ protected function filterProperty(string $property, $direction, QueryBuilder $qu
         }
 
         if (null !== $nullsComparison = $this->properties[$property]['nulls_comparison'] ?? null) {
-            $nullsDirection = self::NULLS_DIRECTION_MAP[$nullsComparison][$direction];
+            if (\in_array($nullsComparison, [self::NULLS_ALWAYS_FIRST, self::NULLS_ALWAYS_LAST], true)) {
+                $nullsDirection = self::NULLS_ALWAYS_FIRST === $nullsComparison ? 'ASC' : 'DESC';
+            } else {
+                $nullsDirection = self::NULLS_DIRECTION_MAP[$nullsComparison][$direction];
+            }
```

Comment:
I think it would be simpler if you reused the direction map instead (clearer and no need to add specific code).

Reply:
Done 👍🏻

Reply:
Do you want to try to fix the MongoDB tests too?

Reply:
I seem to have succeeded. See comment below.